apps/music/js/ui/tab_bar.js

```diff
@@ -44,6 +44,7 @@ var TabBar = {
   init: function tab_init() {
     this.option = 'mix';
     this.view.addEventListener('touchend', this);
+    this.view.addEventListener('click', this);
```

Comment:
@hfiguiere We used to listen to the `click` event but due to https://bugzilla.mozilla.org/show_bug.cgi?id=1043712, we switched to `touchend` events. If we can make sure that issue is no more reproducible, then I think we can just listening to the `click` events and remove the 'touchend' events.